src/pyhf/infer/utils.py

```diff
@@ -9,6 +9,22 @@
 log = logging.getLogger(__name__)
 
 
+def all_pois_floating(pdf, fixed_params):
+    """
+    Checks whether all POI(s) are floatig (i.e. not within the fixed set).
+
+    Args:
+        pdf (:obj:`pyhf.Model`): The model
```

Comment:
```suggestion
        pdf (~pyhf.pdf.Model): The statistical model adhering to the schema ``model.json``.
```
Would be good to match what we have elsewhere.